### PR TITLE
QuickNet-Large

### PR DESCRIPTION
**Pull request by jamescook106**
This PR adds QuickNet. QuickNet is a Larq Compute Engine (https://github.com/larq/compute-engine) optimised Binary ResNet18 with Squeeze and Excite.

### CODE REVIEW

**Comment by AdamHillier** [comment]
Once this is merged it would be good to add automated tests in [larq/compute-engine](https://github.com/larq/compute-engine) which benchmark QuickNet and QuickNetLarge.

**Comment by AdamHillier** [comment]
Or maybe those tests belong in this repo?

**Comment by AdamHillier** [comment]
> cify in PRs that add new models what the accuracy of this model is? I'm trying to verify that the current version of QuickNetLarge indeed corresponds to the 62.50% validation accuracy listed on larq.dev but I'm having a hard time doing so...

I think perhaps in the docstring of the class or a code comment at the top of each model file would work best. Keras applications does that (at least for some models).
